packages/shared/src/components/notifications/Toast.tsx

```diff
@@ -115,7 +107,17 @@ const Toast = ({
           aria-label="Dismiss toast notification"
         />
         {autoDismissNotifications && (
-          <Progress style={{ width: `${progress}%` }} />
+          <div className="flex absolute -bottom-2 left-0 justify-center w-full ease-in-out">
+            <div
+              className={classNames(
+                'h-1 bg-theme-status-cabbage rounded-full transition-all ease-linear',
```

Comment:
I would limit to transition only the width, it can have some negative impacts down the line if we change other variables otherwise.